### ISSUE TITLE
The PromptNER will lead to better understanding of model. But will model generate unspecified tags while evaluating?

### ISSUE THREAD

**Issue by qiuchenyu** [GeorgeLuImmortal/PUnifiedNER#2]

**Comment by GeorgeLuImmortal** [comment]
Exactly, sometimes the model generates entity types that are not included in training set. For example I met once the model generate "[菜品]" for the text span "中国意境菜" which is actually a book.

**Comment by qiuchenyu** [comment]
> Exactly, sometimes the model generates entity types that are not included in training set. For example I met once the model generate "[菜品]" for the text span "中国意境菜" which is actually a book.

So can I understand it as: the meaning of generated type from model may not be accurate, but the extracted texts are basically the entity

**Comment by GeorgeLuImmortal** [comment]
Exactly.
